modules/ROOT/pages/configuring-properties.adoc

```diff
@@ -229,8 +229,9 @@ Configuration properties can be overwritten. The Hierarchy in which these are tr
 . System Properties
 . Environment Properties
 . Deployment Properties
-. Application Properties
+. Application Properties (includes Configuration Properties, Secure Configuration Properties and other Custom Configuration Properties)
+. Global Properties
 
-So, for example, if a configuration property is defined in a System Property, and there is also an application configuration property, the value for that application is the last one.
+So, for example, if a configuration property is defined in a System Property, and there is also an application configuration property, the value for that application is the one with the most precedence (in this case, the System Property).
 
-Also, an application property could depend on Environment, System and/or Deployment properties. Deployment Properties could depend on System Properties, and so on.
+Also, an application property could depend its value on Environment, System and/or Deployment properties. Deployment Properties could depend on System Properties, and so on. For example, if there is a System Property named `env`, a Configuration Property could have its value `file.${env}.xml`. 
```

Comment:
Also, the value of an application property can depend on environment, system, and/or deployment properties. The value of deployment properties can depend on environment and/or system properties and so on. For example, if there is a system property named `env`, a configuration property can have the value `file.${env}.xml`